Generate-Random-number/coverage/lcov.info

```diff
@@ -0,0 +1,193 @@
+TN:
+SF:script.js
```

Comment:
Coverge folder should be in gitignore

Please check gitignore and how it works?
 and add all things whuch are not required into gitignore